cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -64,9 +65,21 @@ spec:
           - name: grafana-persistent-storage
             mountPath: /var
       volumes:
-      - name: influxdb-persistent-storage
-        persistentVolumeClaim:
-          claimName: influxdb-claim
       - name: grafana-persistent-storage
         emptyDir: {}
-  serviceName: monitoring-influxdb
+  volumeClaimTemplates:
+  - metadata:
+      name: influxdb-persistent-storage
```

Comment:
This will show up as a really long pv name `influxdb-persistent-storage-monitoring-influxdb-grafana-v3-0`, maybe we can shorten it?